### ISSUE TITLE
Collection Permission drop-down menu unreadable with dark mode.

### ISSUE THREAD

**Issue by cksapp** [bitwarden/clients#4787]
### Steps To Reproduce

1. Under **Account Settings** -> **Preferences**, change **Theme** to `Dark` and **Save**
2. Within an **Organization** create a new **Collection** or attempt to edit **Access** for an existing collection.
3. When selecting a **User** or **Group** for member access, the **Permissions** drop-down box is shown in white-on-white text within the drop-down, making readability difficult.

### Expected Result

Show drop-down menu with white text on dark background similar to other drop-down menus provided.

### Actual Result

**Collection Permissions** drop-down menu selector options are virtually unreadable and make for difficult usability.

### Screenshots or Videos

![2023-02-16_12h29_46](https://user-images.githubusercontent.com/66083310/219466123-35744191-a435-43c3-ac75-b207885ae781.png)
![2023-02-16_12h30_01](https://user-images.githubusercontent.com/66083310/219466125-4a51c78b-e706-4ee6-aaaa-2db5e49f4479.png)
![2023-02-16_12h30_22](https://user-images.githubusercontent.com/66083310/219466126-1b6b0dba-5019-4fa3-ba64-7940cf9bf079.png)
![2023-02-16_14h17_07](https://user-images.githubusercontent.com/66083310/219466127-98f02c4d-8f89-4152-83ca-a6ac71f88762.png)

### Operating System

Windows

### Web Browser

Chrome, Microsoft Edge

### Build Version

2023.2.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by cksapp** [comment]
Oddly enough this is also prevalent in the **Collection Permission** drop-down menu in the **EDIT MEMBER** view.
Though this shows _correctly_ as expected when initially granting access to a collection from within the **User** edit screen.

![2023-02-16_15h01_56](https://user-images.githubusercontent.com/66083310/219474782-952439f6-5629-4be4-b126-22451b1d3a22.png)

![2023-02-16_15h02_17](https://user-images.githubusercontent.com/66083310/219474784-d4d34675-a54a-4065-8f93-70343539006b.png)

**Comment by dwbit** [comment]
Hey Kent, I just tested this on Firefox, Edge and Chrome and was able to see the drop down items the same way as the type screenshot that you shared.

**Comment by cksapp** [comment]
@dwbit thanks for the info, bummer it doesn't appear consistent :confused:
For some further detail I was able to confirm this across multiple **User** accounts, **Organizations** and types, as well as a new Windows install with consistent visibility issues.

I had not gotten around to checking within Firefox, but this does appear to be a _bit_ better, though still fairly difficult to view IMO.

![2023-02-17_2](https://user-images.githubusercontent.com/66083310/219696117-8fe199e5-8b36-40eb-a1c6-5a3072ab10a7.png)

This appears to be consistent between:

- 3 separate accounts, personal, work, and a test account
- Family, Enterprise, and Free 2-person **Organizations**
- Windows Build
  - Windows 10 Pro<br />22H2 (OS Build 19045.2486)
  - Windows 10 Pro<br />21H2 (OS Build 19044.2604)
- Google Chrome current Version 110.0.5481.104 (Official Build) (64-bit)
-   Microsoft Edge current Version 110.0.1587.49 (Official build) (64-bit)
- Mozilla Firefox current 110.0 (64-bit)

**Comment by dwbit** [comment]
Sorry for the delay, the team was able to confirming this one is hitting Windows machines specifically, and working on a fix! 👍

**Comment by cksapp** [comment]
Awesome, great to hear they were able to confirm the issue here.

I appreciate the follow up and information, looking forward to a fix.

**Comment by MichaelKing1832** [comment]
Confirming issue still unresolved in Firefox 131 as of 2024-10-14.

**Comment by pamperer562580892423** [comment]
@r-tome Regarding this https://github.com/bitwarden/clients/pull/15020 development, I guess this issue can also be closed?!

**Comment by r-tome** [comment]
@pamperer562580892423 Yes, this can be closed — it was fixed in [#15020](https://github.com/bitwarden/clients/pull/15020). Thanks for following up!
